src/components/Overlay/Overlay.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useRef, useEffect} from 'react';
```

Comment:
Please fix the semantics 
![image](https://user-images.githubusercontent.com/54993680/235413788-7c8fd94c-a9ab-40d0-9e49-523f51bd7f89.png)